compose-files/events_handler.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  events-handler:
+    image: oceanprotocol/ocean-events-handler:$EVENTS_HANDLER_VERSION
+    networks:
+      backend:
+        ipv4_address: 172.15.0.22
+    depends_on:
+      - keeper-node
+    environment:
+      KEEPER_URL: ${KEEPER_RPC_URL}
+      PARITY_URL: ${SIGNING_NODE_URL}
+      SECRET_STORE_URL: ${SECRET_STORE_URL}
+      PARITY_ADDRESS: ${PUBLISHER_ADDRESS}
+      PARITY_PASSWORD: ${PUBLISHER_PASSWORD}
+      PARITY_KEYFILE: ${PUBLISHER_KEYFILE}
+      LOCAL_CONTRACTS: ${KEEPER_DEPLOY_CONTRACTS}
+      LOG_LEVEL: ${EVENTS_HANDLER_LOG_LEVEL}
+    volumes:
+      - ${KEEPER_ARTIFACTS_FOLDER}:/usr/local/keeper-contracts:ro
+      - ../networks/spree/authorities/:/parity-data/keys
```

Comment:
i think it must be ${KEEPER_NETWORK} instead of spree, or it will always use spree